packages/docs/developer-resources/integrations/checklist.md

```diff
@@ -0,0 +1,64 @@
+# Checklist
+
+This page serves as a checklist for applications building and integrating on Celo. Using common features and standards across applications in the Celo ecosystem will lead to a superior user experience.
+
+## General
+
+### Addresses
+
+Addresses are identical to Ethereum addresses. When displaying and asking for user-inputted addresses, consider using and validating address checksums following the [EIP55 standard](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md) to detect typos.
```

Comment:
could we add a note here about looking up addresses through the registry rather than hardcoding them?

Reply:
Added a note!